Directory.Build.props

```diff
@@ -69,6 +69,7 @@
     <ExcludeNotSupportedImport>true</ExcludeNotSupportedImport>
     <ExcludePartialFacadesImport>true</ExcludePartialFacadesImport>
     <ExcludeApiCompatImport>true</ExcludeApiCompatImport>
+    <ExcludeReferenceAssembliesImport>true</ExcludeReferenceAssembliesImport>
```

Comment:
```suggestion
    <!-- TODO: Remove when buildtools dependency removed. -->
    <ExcludeReferenceAssembliesImport>true</ExcludeReferenceAssembliesImport>
```

Reply:
Maybe we should add it to the whole PropertyGroup, rather than just this property as the others apply to the same bucket

Reply:
Sounds good.